### PR TITLE
add: server startup password as variable

### PR DESCRIPTION
**Pull request by Panda260**
# Description

you can now start yout teamspeak server with a serverquery password defined under startup

## Checklist for all submissions

* [X] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [X] Have you tested and reviewed your changes with confidence that everything works?
* [X] Did you branch your changes and PR from that branch and not from your master branch?

### CODE REVIEW

File: voice_servers/teamspeak_ARM64/egg-teamspeak3-arm64-server.json [hunk 1]
```diff
@@ -28,6 +28,16 @@
         }
     },
     "variables": [
+        {
+            "name": "Server Query Admin Password",
+            "description": "The password for the server query admin user.",
+            "env_variable": "SERVERADMIN_PASSWORD",
+            "default_value": "changeme",
```
Comment (QuintenQVD0) [inline voice_servers/teamspeak_ARM64/egg-teamspeak3-arm64-server.json:35]: this is a password what means the default value should be empty and it is normaly not required so the rules for this filed need to be `nullable|string|max:32`

File: voice_servers/teamspeak_ARM64/egg-teamspeak3-arm64-server.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
         "ghcr.io\/parkervcp\/yolks:box64": "ghcr.io\/parkervcp\/yolks:box64"
     },
     "file_denylist": [],
-    "startup": "box64 .\/ts3server default_voice_port={{SERVER_PORT}} query_port={{QUERY_PORT}} filetransfer_ip=0.0.0.0 filetransfer_port={{FILE_TRANSFER}} license_accepted=1",
+    "startup": "box64 .\/ts3server default_voice_port={{SERVER_PORT}} query_port={{QUERY_PORT}} filetransfer_ip=0.0.0.0 filetransfer_port={{FILE_TRANSFER}} serveradmin_password={{SERVERADMIN_PASSWORD}} license_accepted=1",
```
Comment (QuintenQVD0) [inline voice_servers/teamspeak_ARM64/egg-teamspeak3-arm64-server.json:16]: Does it work if the SERVERADMIN_PASSWORD variable is empty?

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

File: voice_servers/teamspeak_ARM64/egg-teamspeak3-arm64-server.json (hunk 2 above)
Comment (Panda260) [reply voice_servers/teamspeak_ARM64/egg-teamspeak3-arm64-server.json:16]: in the env stands that is required but if i set it to nullable As you suggested, then the server starts and when you try to connect to yatqa without password you still have no rights
sou i guess its fine

**Comment by QuintenQVD0** [comment]
There should be no default value set for the password

And the variable is empty. And the flag is added to the startup. Can the server still start?

**Comment by Panda260** [comment]
i tested it no problems...

**Review by QuintenQVD0 — APPROVED** [review]

**Comment by QuintenQVD0** [comment]
LGTM! thank you

**Pull request merged**